### DISCUSSION TITLE
[Loading] in fight.def

### DISCUSSION THREAD

**Discussion by doomjoshuaboy** [ikemen-engine/Ikemen-GO#1591] [Feature Request]
### Is your feature request related to a problem? Please describe.

 Its always a pain when it comes to characters like for instance if you have tag with 4 characters each loads slowly depending on the speed of your drives. its better if you have a loading screen in fight def too. 

### Describe the solution you'd like

Similar to like you have  in  [Title info] but with more things 
```
loading.offset = 0, 0
loading.font = 0,0,0
loading.scale = 1, 1
loading.text = "LOADING..."
loading.anim = 0
loading.spr = 0, 0
```

### Additional context

This is used in Nightly Version it just....Nothing. Just waits till the character is loaded with 4v4  Tag Mode:
https://streamable.com/3medg6

### Search terms

Loading, fight ,definition

**Comment by potsmugen** [comment]
I like it. The loading times would be more bearable if the screen informed you of that fact instead of just being all black.
